raft/node.go

```diff
@@ -60,6 +70,10 @@ type Ready struct {
 	// HardState will be equal to empty state if there is no update.
 	pb.HardState
 
+	// The current point-in-time commit info of a Node.
+	// ReadIndexState will be stale if not sending MsgReadIndex first
```

Comment:
// The node can serve linearizable read requests locally when its applied index is greater than the index in ReadState.
// Note that the readState will be returned when raft receives msgReadIndex. The returned readState is only valid for the request.